layouts/default.vue

```diff
@@ -10,10 +10,12 @@ const showUserPicker = logicAnd(
   usePreferences('experimentalUserPicker'),
   () => useUsers().value.length > 1,
 )
+
+const isGrayscale = usePreferences('grayscaleMode')
 </script>
 
 <template>
-  <div h-full>
+  <div h-full :class="{ 'filter-grayscale': isGrayscale, 'filter-none': !isHydrated }">
```

Comment:
What about?

```suggestion
  <div h-full :class="{ 'filter-grayscale': isHydrated && isGrayscale }">
```

Reply:
That was actually my initial implementation 😅 I'll switch back.